ScanbotSDKExamples/NET/ClassicComponent.Droid/MainApplication.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Android.Runtime;
+using Android.Util;
+using DocumentSDK.MAUI.Constants;
+using DocumentSDK.MAUI.Models;
+using DocumentSDK.MAUI.Native.Droid.Utils;
+using NativeDroid = DocumentSDK.MAUI.Native.Droid;
+
+namespace ClassicComponent.Droid
+{
+    // It is strongly recommended to add the LargeHeap = true flag in your Application class.
+    // Working with images, creating PDFs, etc. are memory intensive tasks. So to prevent OutOfMemoryError, consider adding this flag!
+    // For more details see: http://developer.android.com/guide/topics/manifest/application-element.html#largeHeap
+    [Application(LargeHeap = true, Theme = "@style/AppTheme")]
+    public class MainApplication : Application
+    {
+        static readonly string LOG_TAG = typeof(MainApplication).Name;
+
+        // Use a custom temp storage directory for demo purposes.
+        public static TempImageStorageService TempImageStorage;
+
+        // TODO Add the Scanbot SDK license key here.
+        // Please note: The Scanbot SDK will run without a license key for one minute per session!
+        // After the trial period is over all Scanbot SDK functions as well as the UI components will stop working
+        // or may be terminated. You can get an unrestricted "no-strings-attached" 30 day trial license key for free.
+        // Please submit the trial license form (https://scanbot.io/sdk/trial.html) on our website by using
+        // the app identifier "io.scanbot.example.sdk.xamarin" of this example app.
+        const string LICENSE_KEY = null;
```

Comment:
@scan-mayank, same question here regarding the name of the app identifier, should we rather change it to reflect MAUI?

Reply:
Fixed